docetl/operations/resolve.py

```diff
@@ -74,6 +85,22 @@ def compare_pair(
             ):
                 return True, 0
 
+        # For each key that exists in both items, try basic string matching
+        common_keys = set(item1.keys()) & set(item2.keys())
```

Comment:
One worry I have here is that it won't be performant...LLM calls are not performant but at least they can be multithreaded. Do you have an idea about the performance characteristics of slugify?

Reply:
We already check for case-insensitive exact matches in lines 80-86 (in the edited file), so I wonder what the marginal benefit of using slugify is

Reply:
tagging @redhog to give feedback on this, since they raised the issue

Reply:
Fair point, I'll see if I can profile on some of the test cases with it on and off, but most of what I can read online says that the slugify library is pretty efficient.